### PR TITLE
Re-does cosmic heretic lore

### PR DESCRIPTION
**Pull request by Jackal-boop**
## About The Pull Request

rewrites every gain_text of cosmic heretic and its side paths

## Why It's Good For The Game

i really didnt like reading the original and it felt phoned in. please feel free to give suggestions about my writings in the comments

## Changelog
:cl:
spellcheck: re-writes cosmic heretic lore
/:cl:

### CODE REVIEW

File: code/modules/antagonists/heretic/knowledge/side_cosmos_ash.dm [hunk 1]
```diff
@@ -38,7 +38,7 @@
 	desc = "Allows you to transmute a sheet of plasma and a diamond to create an Eldritch Coin. \
 		The coin will open or close nearby doors when landing on heads and bolt or unbolt nearby doors \
 		when landing on tails. If the coin gets inserted into an airlock it emags the door destroying the coin."
-	gain_text = "It tossed the coin and won its bet, now it gains..."
+	gain_text = "The mansus is a place of all sorts of sins. But greed held a special role"
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/side_cosmos_ash.dm:41]:
```suggestion
	gain_text = "The Mansus is a place of all sorts of sins. But greed held a special role."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 2]
```diff
@@ -207,7 +202,7 @@
 	name = "Cosmic Expansion"
 	desc = "Grants you Cosmic Expansion, a spell that creates a 3x3 area of cosmic fields around you. \
 		Nearby beings will also receive a star mark."
-	gain_text = "He was well known, so he had a lot of drawing to do to gather as much of the things he forgot."
+	gain_text = "The ground now shook beneath me. The beast inhabited me, and their voice was intoxicating "
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:205]:
```suggestion
	gain_text = "The ground now shook beneath me. The Beast inhabited me, and their voice was intoxicating."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 3]
```diff
@@ -113,7 +109,7 @@
 	name = "Star Blast"
 	desc = "Fires a projectile that moves very slowly and creates cosmic fields on impact. \
 		Anyone hit by the projectile will recieve burn damage, a knockdown, and give people in a three tile range a star mark."
-	gain_text = "He didn't try, yet felt the call of the night's Creator."
+	gain_text = "The beast was behind me now at all times, with each sacrifice words of affirmation coarsed through me"
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:112]:
```suggestion
	gain_text = "The Beast was behind me now at all times, with each sacrifice words of affirmation coarsed through me."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 4]
```diff
@@ -85,8 +83,7 @@
 	desc = "Your Mansus Grasp now applies the Mark of Cosmos. The mark is triggered from an attack with your Cosmic Blade. \
 		When triggered, the victim is returned to the location where the mark was originally applied to them. \
 		They will then be paralyzed for 2 seconds."
-	gain_text = "As the guide was lost he found a new. The energy increased as the gaze he threw. \
-		He didn't know, but with focus, the Sleepers energy began to flow."
+	gain_text = "The beast now whsipered to me occasionally, only small tidbits of their circumstance's. I can help them, I have to help them."
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:86]:
```suggestion
	gain_text = "The Beast now whispered to me occasionally, only small tidbits of their circumstances. I can help them, I have to help them."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 5]
```diff
@@ -102,8 +99,7 @@
 		will be forced to sleep for 4 seconds. When the victim is hit it also creates a beam that \
 		deals a bit of fire damage and damages the cells. \
 		The beam lasts a minute, until the beam is obstructed or until a new target has been found."
-	gain_text = "He dreamed to know how the matter travelled from star to star. \
-		He lost interest in wanting to find out."
+	gain_text = "After waking in a cold sweat I felt a palm on my scalp, a sigil burned onto me. My veins now emitted a strange purple glow, the beast knows I will surpass its expectations."
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:102]:
```suggestion
	gain_text = "After waking in a cold sweat I felt a palm on my scalp, a sigil burned onto me. My veins now emitted a strange purple glow, the Beast knows I will surpass its expectations."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 6]
```diff
@@ -131,8 +127,7 @@
 		The combo is reset after two seconds without making an attack, \
 		or if you attack someone already marked. If you combo more than four attacks you will recieve, \
 		a cosmic trail and increase your combo timer up to ten seconds."
-	gain_text = "As he ascended to be a watcher, he needed to gather knowledge. \
-		He started to draw it at his home."
+	gain_text = "The beast took my blades in their hand, I kneeled and felt a sharp pain. The blades now glistened with fragmented power. I fell to the ground and wept at the beasts feet"
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:130]:
```suggestion
	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. The blades now glistened with fragmented power. I fell to the ground and wept at the beasts feet."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 7]
```diff
@@ -227,9 +222,7 @@
 		The Star Gazer is a strong ally who can even break down reinforced walls. \
 		The Star Gazer has an aura that will heal you and damage opponents. \
 		Star Touch can now teleport you to the Star Gazer when activated in your hand."
-	gain_text = "The past is gone, the Star Gazer became a vessel to watch over the universe. \
-		The Creator made this his path and he forgot his purpose. \
-		THE TIME IS NOW, WITNESS MY ASCENSION, THE STAR GAZER HAS GAINED PURPOSE ONCE MORE!"
+	gain_text = "The beast held out its hand, I grabbed hold and they pulled me to them. Their body was towering, but it seemed so small and feeble after all their tales compiled in my head. I clung on to them, they would protect me, and I would protect it. I closed my eyes with my head laid against their form. I was safe. WITTNESS MY ASCENTION"
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:225]:
```suggestion
	gain_text = "The Beast held out its hand, I grabbed hold and they pulled me to them. Their body was towering, but it seemed so small and feeble after all their tales compiled in my head. I clung on to them, they would protect me, and I would protect it. I closed my eyes with my head laid against their form. I was safe. WITNESS MY ASCENSION!"
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 8]
```diff
@@ -31,7 +31,7 @@
 	desc = "Opens up the Path of Cosmos to you. \
 		Allows you to transmute a sheet of plasma and a knife into an Cosmic Blade. \
 		You can only create two at a time."
-	gain_text = "It looked at the stars to guide himself."
+	gain_text = "A nebula appeared in the sky, its infernal births shone onto me, this was the start of a great transcendence."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:34]:
```suggestion
	gain_text = "A nebula appeared in the sky, its infernal birth shone upon me. This was the start of a great transcendence."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 9]
```diff
@@ -43,8 +43,7 @@
 /datum/heretic_knowledge/cosmic_grasp
 	name = "Grasp of Cosmos"
 	desc = "Your Mansus Grasp will give people a star mark (cosmic ring) and create a cosmic field where you stand."
-	gain_text = "The more he looked the more everything made sense. \
-		The stars traced out the path forward to his home."
+	gain_text = "Some stars dimmed, other's magnitude increased. With newfound strength I could channel the nebula's power into myself."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:46]:
```suggestion
	gain_text = "Some stars dimmed, others' magnitude increased. With newfound strength I could channel the nebula's power into myself."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 10]
```diff
@@ -68,8 +67,7 @@
 	desc = "Grants you Cosmic Runes, a spell that creates two runes linked with eachother for easy teleportation. \
 		Only the entity activating the rune will get transported, and it can be used by anyone without a star mark. \
 		However, people with a star mark will get transported along with another person using the rune."
-	gain_text = "When day came, the Sleeper got lost. \
-		The sun outshone the stars, so he lost his guide."
+	gain_text = "A sigil appeared below me, I lowered my face to it and heard distant roars, I flung myself in and saw a beast behind dense clouds of gas."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:70]:
```suggestion
	gain_text = "A sigil appeared below me, I lowered my face to it and heard distant roars. I flung myself in and saw a beast behind dense clouds of gas."
```
I'll be honest I also think the repeated use of the word "beast" here is kind of uninspired and not very mystical. Can we call it something else? 
Even just a "figure" is more interesting to me than "beast".

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 8 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:34]:
```suggestion
	gain_text = "A new nebula burned in the sky, its infernal light blazed down upon me. This was the start of a great transcendence."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 9 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:46]:
```suggestion
	gain_text = "The nebula boiled and churned in turmoil. I reached out and pulled the chaos into my heart."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 10 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:70]:
```suggestion
	gain_text = "The distant stars crept into my dreams, roaring and screaming without reason. I spoke, and heard my own words echoed back."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 4 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:86]:
```suggestion
	gain_text = "Their halting whispers fill my ears, growing clearer every day. I can help them, I have to help them."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 5 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:102]:
```suggestion
	gain_text = "Having learned to speak, they wanted to feel. I invited them into my veins. The stars burn through me with every pulse."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 3 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:112]:
```suggestion
	gain_text = "What they know, I know. What I know, they know. The stars are coming and they are becoming, they promise that we will become together."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 6 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:130]:
```suggestion
	gain_text = "When I falter, their gravity pulls me back up. They move my limbs before I know what it is I am grasping for. Nothing we strive for together is beyond our reach."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 2 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:205]:
```suggestion
	gain_text = "Where they are is cold and empty, but here we burn with life. Their regard crowds out my thoughts, their power is only a whisper away. With each tug, they come closer."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 7 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:225]:
```suggestion
	gain_text = "We grasped hands across time, bridging the unimaginable gap, and pulled until we met. They learned their shape from me and it is beautiful. I closed our eyes, and laid my head upon the expanse of our chest. At last I am the stars, and the stars me too. We cannot be separated, WITNESS OUR ASCENSION."
```

File: code/modules/antagonists/heretic/knowledge/side_cosmos_ash.dm [hunk 11]
```diff
@@ -5,7 +5,7 @@
 	desc = "Allows you to transmute a pool of ash, a liver, and a sheet of plasma into a Fire Shark. \
 		Fire Sharks are fast and strong in groups, but die quickly. They are also highly resistant against fire attacks. \
 		Fire Sharks inject phlogiston into its victims and spawn plasma once they die."
-	gain_text = "My knowledge of the universe with the energy of remains, constructed it. It gave the Fire Shark life."
+	gain_text = "The nebula teemed with frenzied creatures, one of which emitted a dank purple smoke."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/side_cosmos_ash.dm:8]:
```suggestion
	gain_text = "The cradle of the nebula was cold, but not dead. Light and heat flits even through the deepest darkness, and is hunted by its own predators."
```

File: code/modules/antagonists/heretic/knowledge/side_cosmos_ash.dm (hunk 1 above)
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/side_cosmos_ash.dm:41]:
```suggestion
	gain_text = "They were fascinated by chance, probability, the same action providing different results. But they don't play fair."
```

**Review by Jacquerel — COMMENTED** [review]
Non-reviews, but you asked for suggestions. 
You can ignore as much of this as you want but while the above is "grammatical stuff you should probably fix" this is me trying to portray a similar narrative but in a way I think is cooler and has a little bit more spooky flavouring.

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 10 above)
Comment (Jackal-boop) [reply code/modules/antagonists/heretic/knowledge/cosmic_lore.dm]: i really like this suggestion

**Comment by Jackal-boop** [comment]
okay i picked out what i really liked from jaquarels suggestions, thank you jaquarel

**Comment by Jackal-boop** [comment]
also i forgot to mention this in the PR body but this is also kinda supposed to be a love story.

**Comment by Jacquerel** [comment]
> also i forgot to mention this in the PR body but this is also kinda supposed to be a love story.

That's good because when I was contributing I also started to think "hmm what I'm writing sounds sort of romantic" and then didn't change it 😅

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 12]
```diff
@@ -113,7 +109,7 @@
 	name = "Star Blast"
 	desc = "Fires a projectile that moves very slowly and creates cosmic fields on impact. \
 		Anyone hit by the projectile will recieve burn damage, a knockdown, and give people in a three tile range a star mark."
-	gain_text = "He didn't try, yet felt the call of the night's Creator."
+	gain_text = "The Beast was behind me now at all times, with each sacrifice words of affirmation coarsed through me."
```
Comment (wulfenbach-jpg) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:112]:
```suggestion
	gain_text = "The Beast was behind me now at all times, with each sacrifice words of affirmation coursed through me."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 13]
```diff
@@ -131,8 +127,7 @@
 		The combo is reset after two seconds without making an attack, \
 		or if you attack someone already marked. If you combo more than four attacks you will recieve, \
 		a cosmic trail and increase your combo timer up to ten seconds."
-	gain_text = "As he ascended to be a watcher, he needed to gather knowledge. \
-		He started to draw it at his home."
+	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. The blades now glistened with fragmented power. I fell to the ground and wept at the beasts feet."
```
Comment (wulfenbach-jpg) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:130]:
```suggestion
	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. The blades now glistened with fragmented power. I fell to the ground and wept at the beast's feet."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 14]
```diff
@@ -227,9 +222,7 @@
 		The Star Gazer is a strong ally who can even break down reinforced walls. \
 		The Star Gazer has an aura that will heal you and damage opponents. \
 		Star Touch can now teleport you to the Star Gazer when activated in your hand."
-	gain_text = "The past is gone, the Star Gazer became a vessel to watch over the universe. \
-		The Creator made this his path and he forgot his purpose. \
-		THE TIME IS NOW, WITNESS MY ASCENSION, THE STAR GAZER HAS GAINED PURPOSE ONCE MORE!"
+	gain_text = "The Beast held out its hand, I grabbed hold and they pulled me to them. Their body was towering, but it seemed so small and feeble after all their tales compiled in my head. I clung on to them, they would protect me, and I would protect it. I closed my eyes with my head laid against their form. I was safe. WITNESS MY ASCENSION!"
```
Comment (wulfenbach-jpg) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:225]:
```suggestion
	gain_text = "The Beast held out its hand, I grabbed ahold and they pulled me to them. Their body was towering, but it seemed so small and feeble after all their tales compiled in my head. I clung on to them, they would protect me, and I would protect it. I closed my eyes with my head laid against their form. I was safe. WITNESS MY ASCENSION!"
```

**Review by wulfenbach-jpg — CHANGES_REQUESTED** [review]
Just a few grammatical nitpicks.

**Review by Fikou — CHANGES_REQUESTED** [review]
use line splitters like the old lore did

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm (hunk 13 above)
Comment (MrMelbert) [reply code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:130]:
```suggestion
	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. The blades now glistened with fragmented power. I fell to the ground and wept at the Beast's feet."
```

**Comment by Jackal-boop** [comment]
alrighty there we go

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 15]
```diff
@@ -131,8 +127,7 @@
 		The combo is reset after two seconds without making an attack, \
 		or if you attack someone already marked. If you combo more than four attacks you will recieve, \
 		a cosmic trail and increase your combo timer up to ten seconds."
-	gain_text = "As he ascended to be a watcher, he needed to gather knowledge. \
-		He started to draw it at his home."
+	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. \ The blades now glistened with fragmented power. I fell to the ground and wept at the beast's feet."
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:130]: You have to actually split the line if you put in a line splitter
```suggestion
	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. \ 
		The blades now glistened with fragmented power. I fell to the ground and wept at the beast's feet."
```
Not going through all of these but yeah

Comment (Jackal-boop) [reply code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:130]: Oh I'm stupid I thought the slash did that

**Comment by Jackal-boop** [comment]
This is ready

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 16]
```diff
@@ -43,8 +43,8 @@
 /datum/heretic_knowledge/cosmic_grasp
 	name = "Grasp of Cosmos"
 	desc = "Your Mansus Grasp will give people a star mark (cosmic ring) and create a cosmic field where you stand."
-	gain_text = "The more he looked the more everything made sense. \
-		The stars traced out the path forward to his home."
+	gain_text = "Some stars dimmed, others' magnitude increased. \
+	With newfound strength I could channel the nebula's power into myself."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:47]:
```suggestion
		With newfound strength I could channel the nebula's power into myself."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 17]
```diff
@@ -68,8 +68,8 @@
 	desc = "Grants you Cosmic Runes, a spell that creates two runes linked with eachother for easy teleportation. \
 		Only the entity activating the rune will get transported, and it can be used by anyone without a star mark. \
 		However, people with a star mark will get transported along with another person using the rune."
-	gain_text = "When day came, the Sleeper got lost. \
-		The sun outshone the stars, so he lost his guide."
+	gain_text = "The distant stars crept into my dreams, roaring and screaming without reason. \
+	I spoke, and heard my own words echoed back."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:72]:
```suggestion
		I spoke, and heard my own words echoed back."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 18]
```diff
@@ -85,8 +85,8 @@
 	desc = "Your Mansus Grasp now applies the Mark of Cosmos. The mark is triggered from an attack with your Cosmic Blade. \
 		When triggered, the victim is returned to the location where the mark was originally applied to them. \
 		They will then be paralyzed for 2 seconds."
-	gain_text = "As the guide was lost he found a new. The energy increased as the gaze he threw. \
-		He didn't know, but with focus, the Sleepers energy began to flow."
+	gain_text = "The Beast now whispered to me occasionally, only small tidbits of their circumstances. \
+	I can help them, I have to help them."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:89]:
```suggestion
		I can help them, I have to help them."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 19]
```diff
@@ -102,8 +102,8 @@
 		will be forced to sleep for 4 seconds. When the victim is hit it also creates a beam that \
 		deals a bit of fire damage and damages the cells. \
 		The beam lasts a minute, until the beam is obstructed or until a new target has been found."
-	gain_text = "He dreamed to know how the matter travelled from star to star. \
-		He lost interest in wanting to find out."
+	gain_text = "After waking in a cold sweat I felt a palm on my scalp, a sigil burned onto me. \
+	My veins now emitted a strange purple glow, the Beast knows I will surpass its expectations."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:106]:
```suggestion
		My veins now emitted a strange purple glow, the Beast knows I will surpass its expectations."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 20]
```diff
@@ -131,8 +131,8 @@
 		The combo is reset after two seconds without making an attack, \
 		or if you attack someone already marked. If you combo more than four attacks you will recieve, \
 		a cosmic trail and increase your combo timer up to ten seconds."
-	gain_text = "As he ascended to be a watcher, he needed to gather knowledge. \
-		He started to draw it at his home."
+	gain_text = "The Beast took my blades in their hand, I kneeled and felt a sharp pain. \
+	The blades now glistened with fragmented power. I fell to the ground and wept at the beast's feet."
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:135]:
```suggestion
		The blades now glistened with fragmented power. I fell to the ground and wept at the beast's feet."
```

File: code/modules/antagonists/heretic/knowledge/cosmic_lore.dm [hunk 21]
```diff
@@ -227,9 +227,10 @@
 		The Star Gazer is a strong ally who can even break down reinforced walls. \
 		The Star Gazer has an aura that will heal you and damage opponents. \
 		Star Touch can now teleport you to the Star Gazer when activated in your hand."
-	gain_text = "The past is gone, the Star Gazer became a vessel to watch over the universe. \
-		The Creator made this his path and he forgot his purpose. \
-		THE TIME IS NOW, WITNESS MY ASCENSION, THE STAR GAZER HAS GAINED PURPOSE ONCE MORE!"
+	gain_text = "The Beast held out its hand, I grabbed hold and they pulled me to them. Their body was towering, but it seemed so small and feeble after all their tales compiled in my head. \
+	I clung on to them, they would protect me, and I would protect it. \
+	I closed my eyes with my head laid against their form. I was safe. \
+	WITNESS MY ASCENSION!"
```
Comment (Jacquerel) [inline code/modules/antagonists/heretic/knowledge/cosmic_lore.dm:233]:
```suggestion
		I clung on to them, they would protect me, and I would protect it. \
		I closed my eyes with my head laid against their form. I was safe. \
		WITNESS MY ASCENSION!"
```

**Review by Jacquerel — CHANGES_REQUESTED** [review]
Indentation is off here.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**